jax/experimental/maps.py

```diff
@@ -384,8 +384,8 @@ def xmap(fun: Callable,
     def regression_loss(x, y, w, b):
       # Contract over in_features. Batch and out_features are present in
       # both inputs and output, so they don't need to be mentioned
-      y_pred = jnp.einsum('{in_features},{in_features}->{}') + b
-      return jnp.mean((y - y_pred) ** 2, axis='batch')
+      y_pred = jnp.einsum('{in_features},{in_features}->{}', x, w) + b
+      return jnp.mean((y - y_pred) ** 2, axis=('batch', 'out_features'))
```

Comment:
```suggestion
      error = jnp.sum((y - y_pred) ** 2, axis='out_features')
      return jnp.mean(error, axis='batch')
```

I think we want a sum to compute the l2 norm, then a mean over the batch, right?

Reply:
Right, of course! 🤦‍♂️